src/main/java/frc/team2412/robot/commands/intake/IntakeDiagnosticCommand.java

```diff
@@ -0,0 +1,16 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+
+public class IntakeDiagnosticCommand extends SequentialCommandGroup {
+
+    private final IntakeSubsystem subsystem;
+
+    public IntakeDiagnosticCommand(IntakeSubsystem subsystem) {
+        this.subsystem = subsystem;
+        addRequirements(subsystem);
+        addCommands(new IntakeExtendCommand(subsystem), new IntakeInCommand(subsystem),
+                new IntakeOutCommand(subsystem), new IntakeStopCommand(subsystem), new IntakeRetractCommand(subsystem));
```

Comment:
Best to add waits between the commands to make it easy to watch the robot and see the command affect it. As written, the whole command sequence will finish in about a tenth of a second (0.02s loop iteration time * five commands).  Use a WaitCommand(<time>) to do the waits